docs/development/dev-services.rst

```diff
@@ -203,9 +203,9 @@ Install dependencies
   sudo pip-2.7 install docutils
   brew install mercurial
 
-Install lessc
+Install node dependencies
 ::
-  sudo npm install -g less
+  npm install less
```

Comment:
But you're right. We don't need to explictly say these instructions because `webapp-django/bin/bootstrap.sh` *does* indeed take care of that.